src/utils/mediafetch/bililist.ts

```
@@ -0,0 +1,78 @@
+      ...js.data.items_lists.seasons_list,
+      ...js.data.items_lists.series_list,
+    ],
+    getBVID: v => v,
+  }) as Promise<any[]>;
+};
+
+// HACK: its only the top 5 videos from the list
+export const getBVIDFast = async (mid: string) => {
+  const list = await fetchLists(mid);
+  return list.reduce(
+    (acc, curr) => [...acc, ...curr.archives.map((v: any) => v.bvid)],
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)